docs/alerts/alertmanager.yml

```diff
@@ -0,0 +1,81 @@
+global:
+  # The smarthost and SMTP sender used for mail notifications.
+  # we are using gmail SMTP in this example
+  smtp_smarthost: 'smtp.gmail.com:587'
+  smtp_from: 'suraj.sidh+alertmanager@grafana.com'
+  smtp_auth_username: 'suraj.sidh@grafana.com'
+  smtp_auth_identity: 'suraj.sidh@grafana.com'
+  smtp_auth_password: '<gmail-password>'
+
+# The directory from which notification templates are read.
+templates:
+  - '/etc/alertmanager/template/*.tmpl'
+
+# The root route on which each incoming alert enters.
+route:
+  # The labels by which incoming alerts are grouped together. For example,
+  # multiple alerts coming in for cluster=A and alertname=LatencyHigh would
+  # be batched into a single group.
+  #
+  # To aggregate by all possible labels use '...' as the sole label name.
+  # This effectively disables aggregation entirely, passing through all
+  # alerts as-is. This is unlikely to be what you want, unless you have
+  # a very low alert volume or your upstream notification system performs
+  # its own grouping. Example: group_by: [...]
+  group_by: ['alertname', 'cluster', 'service']
+
+  # When a new group of alerts is created by an incoming alert, wait at
+  # least 'group_wait' to send the initial notification.
+  # This way ensures that you get multiple alerts for the same group that start
+  # firing shortly after another are batched together on the first
+  # notification.
+  group_wait: 5m
+
+  # When the first notification was sent, wait 'group_interval' to send a batch
+  # of new alerts that started firing for that group.
+  group_interval: 15m
+
+  # If an alert has successfully been sent, wait 'repeat_interval' to
+  # resend them.
+  repeat_interval: 2h
+
+  # A default receiver
+  receiver: suraj-email
+
+  # All the above attributes are inherited by all child routes and can
+  # overwritten on each.
+
+  # The child route trees.
+  routes:
+    # This routes performs a regular expression match on alert labels to
+    # catch alerts that are related to a list of services.
+    - match_re:
+        job: ^(website)$
+      receiver: suraj-sm-email
+
+# Inhibition rules allow to mute a set of alerts given that another alert is
+# firing.
+# We use this to mute any warning-level notifications if the same alert is
+# already critical.
+inhibit_rules:
+  - source_match:
+      severity: 'critical'
+    target_match:
+      severity: 'warning'
+    # Apply inhibition if the alertname is the same.
+    # CAUTION:
+    #   If all label names listed in `equal` are missing
+    #   from both the source and target alerts,
+    #   the inhibition rule will apply!
+    equal: ['alertname', 'cluster', 'service']
+
+
+receivers:
+  - name: 'suraj-email'
+    email_configs:
+      - to: 'suraj.sidh+alerts@grafana.com'
```

Comment:
I think if you put an actual email in here, there is a very possibility you will get a bunch of folks alerts when they are giving this a try

Reply:
makes sense, updated